validation_test/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.7-slim
+
+WORKDIR /service
+
+RUN pip install mlflow==1.17.0
```

Comment:
would be nice to have a simple `pyproiect.toml` to deal with the dependencies and the basic usage. 

Also is there a need for the docker image? Isn't it simpler to run everything directly on the host?